core/templates/dev/head/components/state-editor/state-content-editor/state-content-editor.directive.ts

```diff
@@ -48,11 +48,11 @@ angular.module('oppia').directive('stateContentEditor', [
         '/components/state-editor/state-content-editor/' +
         'state-content-editor.directive.html'),
       controller: [
-        '$scope', 'StateContentService', 'EditabilityService',
-        'EditorFirstTimeEventsService',
+        '$scope', 'StateContentService', 'StateEditorService',
+        'EditabilityService', 'EditorFirstTimeEventsService',
```

Comment:
Can you add this in sorted order? (Considering this wasn't in order in the first place.)

Reply:
Done.